c7n/resources/swf.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n import tags
+
+
+@resources.register('swf')
```

Comment:
s/swf/swf-domain  ... afaics the domain is just a namespace

Reply:
keeping it as `swf` and updating class name to `SimpleWorkflow` instead of `SimpleWorkflowDomain` for consistency

Reply:
i'm still unclear why this should be called swf vs swf-domain.. its not a workflow. its a namespace container.. ie we don't have 'ecs' for an 'ecs-cluster'

Reply:
Ahh sorry, didn't understand your first comment but @PratMis clarified, just pushed up changes